docs/data/migrate-from-horizon-to-rpc/README.mdx

```diff
@@ -0,0 +1,123 @@
+
+## Request / Response Format
+
+Horizon's REST-like API uses HTTP methods and status codes to communicate with clients. Responses are JSON in the HAL format. See [Horizon's Response Format].
+
+RPC's JSON-RPC API uses JSON-RPC 2.0 to communicate with clients. Requests to the API are JSON objects that contain one or more method invocations. Responses are also JSON objects that contain a result for each invocation in the request. See [JSON-RPC].
+
+Both formats utilise JSON for the overall structure which are relatively simple and do not require any special client code, although there are client [SDKs] available. Some values contained within are XDR encoded and can be decoded using Stellar [SDKs].
+
+## Endpoint Mapping
+
+Applications that use the following Horizon endpoints can typically migrate directly to the RPC using the referenced methods.
+
+Endpoints without mappings do not have a direct replacement in the RPC API. To build similar functionality in an application, take a look at other [Data] products.
+
+| Horizon Endpoint | RPC Method |
+| --- | --- |
+| [`GET /`] | [`getLatestLedger`] [`getVersionInfo`] [`getHealth`] [`getNetwork`] |
+| [`GET /ledgers`] | [`getLedgers`] |
+| [`GET /ledgers/{seq}`] | [`getLedgers`] |
+| [`GET /ledgers/{seq}/transactions`] | [`getTransactions`] |
+| [`GET /ledgers/{seq}/operations`] | [`getTransactions`] |
+| [`GET /ledgers/{seq}/payments`] | [`getEvents`] ⚠️ |
+| [`GET /ledgers/{seq}/effects`] | [`getEvents`] ⚠️ |
+| [`POST /transactions`] | [`sendTransaction`] |
+| [`POST /transactions_async`] | [`sendTransaction`] |
+| [`GET /transactions`] | [`getTransactions`] |
+| [`GET /transactions/{hash}`] | [`getTransaction`] |
+| [`GET /transactions/{hash}/operations`] | [`getTransaction`] |
+| [`GET /transactions/{hash}/payments`] | - |
+| [`GET /transactions/{hash}/effects`] | - |
+| [`GET /operations`] | [`getTransactions`] |
+| [`GET /operations/{id}`] | - |
+| [`GET /operations/{id}/effects`] | - |
+| [`GET /fee_stats`] | [`getFeeStats`] [`simulateTransaction`] |
+| [`GET /accounts`] | - |
+| [`GET /accounts/{address}`] | [`getLedgerEntries`] |
+| [`GET /claimable_balances`] | - |
+| [`GET /claimable_balances/{id}`] | [`getLedgerEntries`] |
+| [`GET /claimable_balances/{id}/transactions`] | - |
+| [`GET /claimable_balances/{id}/operations`] | - |
+| [`GET /liquidity_pools`] | - |
+| [`GET /liquidity_pools/{id}`] | [`getLedgerEntries`] |
+| [`GET /liquidity_pools/{id}/transactions`] | - |
+| [`GET /liquidity_pools/{id}/operations`] | - |
+| [`GET /liquidity_pools/{id}/effects`] | - |
+| [`GET /liquidity_pools/{id}/trades`] | - |
+| [`GET /offers`] | - |
+| [`GET /offers/{id}`] | [`getLedgerEntries`] |
+| [`GET /offers/{id}/trades`] | - |
+| [`GET /payments`] | [`getEvents`] ⚠️ |
+| [`GET /effects`] | [`getEvents`] ⚠️ |
+| [`GET /trades`] | [`getEvents`] ⚠️ |
```

Comment:
Similar comment to above, regarding the combo of `getTransactions` + `getEvents`

Reply:
7ee00763a9aec54b0750c28982ac991098271352